mediapipe/calculators/ovms/openvinoinferencecalculator.cc

```diff
@@ -333,12 +356,88 @@ class OpenVINOInferenceCalculator : public CalculatorBase {
     std::unique_ptr<tflite::Interpreter> interpreter_ = absl::make_unique<tflite::Interpreter>();
     bool initialized = false;
 
+    static bool ValidateTagToNames(CalculatorContract* cc)
```

Comment:
This function reminds me of
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#indentation
Please split this up.